fuel/src/main/kotlin/com/github/kittinunf/fuel/core/FuelError.kt

```diff
@@ -7,7 +7,7 @@ private typealias StackTrace = Array<out StackTraceElement>
 /**
  * Indicates a call to [FuelError.wrap] passing in a [FuelError]
  */
-private class BubbleFuelError(inner: FuelError) : FuelError(inner, inner.response)
+private class BubbleFuelError(val inner: FuelError) : FuelError(inner, inner.response)
```

Comment:
Do we really need to store it? Isn't that the same as (line 84) -> it.super or something like that?

Reply:
Without this, we couldn't access the inner `FuelError` right?

Reply:
Well, it's the parent of the Bubbel, is it not :P ? I'm uncertain ha

Reply:
🤔 something like this? ffd9fbc

Reply:
Yeah. I liked your previous code better. Let's just store it as inner. It's much more clear.

Reply:
Done sir! @SleeplessByte